netket/experimental/operator/_fermions_2nd.py

```diff
@@ -0,0 +1,605 @@
+from multiprocessing.sharedctypes import Value
+from typing import List, Union
+from netket.utils.types import DType
+
+import numpy as np
+from numba import jit
+
+from netket.hilbert import AbstractHilbert
+from netket.experimental.hilbert import SpinOrbitalFermions
+
+from netket.operator._discrete_operator import DiscreteOperator
+from netket.operator._pauli_strings import _count_of_locations
+from netket.hilbert.abstract_hilbert import AbstractHilbert
+import re
+from collections import defaultdict
```

Comment:
why do you need this?

Reply:
_count_of_locations is something that looks for the number of qubits/fermions/etc in openfermion operators